tasks/go.py

```diff
@@ -391,6 +411,40 @@ def is_excluded(pkg):
                         print("Skipping {} ({}) excluded in .wwhrd.yml".format(package, license))
                     else:
                         licenses.append("core,\"{}\",{}".format(package, license))
+
+    # Additional Licenses
+    for pkg in additional:
+        if pkg.startswith(GITHUB_DOMAIN):
+            url = "https://{}/master/".format(pkg.replace(GITHUB_DOMAIN, GITHUB_RAW_DOMAIN))
+        else:
+            url = "https://{}".format(pkg)
+
+        for filename in LICENSE_FILES:
+
+            url = urljoin(url, filename)
+            try:
+                resp = requests.get(url)
+                resp.raise_for_status()
+
+                with tempfile.TemporaryDirectory() as tempdir:
+                    with open(os.path.join(tempdir, filename), 'w') as lfp:
+                        lfp.write(resp.text)
+                        lfp.flush()
+
+                        temp_path = os.path.dirname(lfp.name)
+                        result = ctx.run("go run github.com/go-enry/go-license-detector/v4/cmd/license-detector -f json {}".format(temp_path), hide='err')
```

Comment:
Do we need to `hide='err'` here? It sounds useful to have the errors in the job log to troubleshoot